File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/PackagesPackage.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class PackagesPackage {
    @JsonProperty("ActualDepartureDateTime")
    public String actualDepartureDateTime;
    @JsonProperty("ExternalId")
    public Integer externalId;
    @JsonProperty("GrossUnitOfWeightAbbreviation")
    public String grossUnitOfWeightAbbreviation;
    @JsonProperty("GrossWeight")
    public Double grossWeight;
    @JsonProperty("Id")
    public Integer id;
    @JsonProperty("ItemStrainName")
    public String itemStrainName;
    @JsonProperty("LabTestingStateName")
    public String labTestingStateName;
    @JsonProperty("ManifestNumber")
    public String manifestNumber;
    @JsonProperty("PackageId")
    public Integer packageId;
    @JsonProperty("PackageLabel")
    public String packageLabel;
    @JsonProperty("ProcessingJobTypeName")
    public String processingJobTypeName;
    @JsonProperty("ProductCategoryName")
    public String productCategoryName;
    @JsonProperty("ProductName")
    public String productName;
    @JsonProperty("ReceivedDateTime")
    public String receivedDateTime;
    @JsonProperty("ReceivedQuantity")
    public Double receivedQuantity;
    @JsonProperty("ReceivedUnitOfMeasureAbbreviation")
    public String receivedUnitOfMeasureAbbreviation;
    @JsonProperty("ReceiverWholesalePrice")
    public Double receiverWholesalePrice;
    @JsonProperty("RecipientFacilityLicenseNumber")
    public String recipientFacilityLicenseNumber;
    @JsonProperty("RecipientFacilityName")
    public String recipientFacilityName;
    @JsonProperty("ShipmentPackageStateName")
    public String shipmentPackageStateName;
    @JsonProperty("ShippedQuantity")
    public Double shippedQuantity;
    @JsonProperty("ShippedUnitOfMeasureAbbreviation")
    public String shippedUnitOfMeasureAbbreviation;
    @JsonProperty("ShipperWholesalePrice")
    public Double shipperWholesalePrice;
    @JsonProperty("SourceHarvestNames")
    public String sourceHarvestNames;
    @JsonProperty("SourcePackageLabels")
    public String sourcePackageLabels;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PackagesPackage.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PackagesPackage(
    @JsonProperty("ActualDepartureDateTime")
    val actualDepartureDateTime: String? = null,
    @JsonProperty("ExternalId")
    val externalId: Int? = null,
    @JsonProperty("GrossUnitOfWeightAbbreviation")
    val grossUnitOfWeightAbbreviation: String? = null,
    @JsonProperty("GrossWeight")
    val grossWeight: Double? = null,
    @JsonProperty("Id")
    val id: Int? = null,
    @JsonProperty("ItemStrainName")
    val itemStrainName: String? = null,
    @JsonProperty("LabTestingStateName")
    val labTestingStateName: String? = null,
    @JsonProperty("ManifestNumber")
    val manifestNumber: String? = null,
    @JsonProperty("PackageId")
    val packageId: Int? = null,
    @JsonProperty("PackageLabel")
    val packageLabel: String? = null,
    @JsonProperty("ProcessingJobTypeName")
    val processingJobTypeName: String? = null,
    @JsonProperty("ProductCategoryName")
    val productCategoryName: String? = null,
    @JsonProperty("ProductName")
    val productName: String? = null,
    @JsonProperty("ReceivedDateTime")
    val receivedDateTime: String? = null,
    @JsonProperty("ReceivedQuantity")
    val receivedQuantity: Double? = null,
    @JsonProperty("ReceivedUnitOfMeasureAbbreviation")
    val receivedUnitOfMeasureAbbreviation: String? = null,
    @JsonProperty("ReceiverWholesalePrice")
    val receiverWholesalePrice: Double? = null,
    @JsonProperty("RecipientFacilityLicenseNumber")
    val recipientFacilityLicenseNumber: String? = null,
    @JsonProperty("RecipientFacilityName")
    val recipientFacilityName: String? = null,
    @JsonProperty("ShipmentPackageStateName")
    val shipmentPackageStateName: String? = null,
    @JsonProperty("ShippedQuantity")
    val shippedQuantity: Double? = null,
    @JsonProperty("ShippedUnitOfMeasureAbbreviation")
    val shippedUnitOfMeasureAbbreviation: String? = null,
    @JsonProperty("ShipperWholesalePrice")
    val shipperWholesalePrice: Double? = null,
    @JsonProperty("SourceHarvestNames")
    val sourceHarvestNames: String? = null,
    @JsonProperty("SourcePackageLabels")
    val sourcePackageLabels: String? = null
)
